lib/mix/lib/mix/tasks/profile.tprof.ex

```diff
@@ -267,6 +268,9 @@ defmodule Mix.Tasks.Profile.Tprof do
         :calls ->
           :calls
 
+        :per_call ->
+          :measurement_per_call
```

Comment:
Note: we could add a `when type != :calls` here because it doesn't really make sense and would just return arbitrary order, but since `tprof` allows it I thought there is no need to be extra defensive.